Jellyfin.Plugin.TMDbBoxSets/Jellyfin.Plugin.TMDbBoxSets.csproj

```diff
@@ -3,8 +3,6 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <IsPackable>true</IsPackable>
-    <AssemblyVersion>2.0.0</AssemblyVersion>
```

Comment:
We can't remove these or the plugins break in unexpected ways.

Reply:
IIRC This actually sets the viewable plugin version in the software, so it should be increments as well? @joshuaboniface Does the catalogue read this version?

Reply:
The catalog doesn't, but once installed it shows as the plugin version.